src/redisboard/views.py

```diff
@@ -42,10 +42,17 @@ def _get_key_info(conn, key):
     try:
         obj_type = conn.type(key)
         pipe = conn.pipeline()
-        pipe.execute_command('DEBUG', 'OBJECT', key)
-        LENGTH_GETTERS[obj_type](pipe, key)
-        pipe.ttl(key)
-        details, obj_length, obj_ttl = pipe.execute()
+        try:
+            pipe.execute_command('DEBUG', 'OBJECT', key)
+            LENGTH_GETTERS[obj_type](pipe, key)
+            pipe.ttl(key)
+            details, obj_length, obj_ttl = pipe.execute()
+        except ResponseError as exc:
+            logger.exception(exc)
```

Comment:
One small nit: should log something like `logger.exception("Failed to get object info for key %r: %s", key, exc)` 

That will help people debug failures (in case there's something wrong with their redis server or something).